packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -654,6 +679,17 @@ export const DEFAULT_INTERMEDIATE_TOKENS_BY_CHAIN_ID = valueByChainId<string[]>(
             AVALANCHE_TOKENS.USDT,
             AVALANCHE_TOKENS.USDC,
         ],
+        [ChainId.Celo]: [
+            CELO_TOKENS.cUSD,
+            CELO_TOKENS.cEUR,
+            CELO_TOKENS.cETH,
+            CELO_TOKENS.cBTC,
+            CELO_TOKENS.mCUSD,
+            CELO_TOKENS.mCEUR,
+            CELO_TOKENS.UBE,
+            CELO_TOKENS.WCELO,
+            CELO_TOKENS.mCELO,
```

Comment:
Based on the pairs that have liquidity on Ubeswap and Sushiswap, I think we can pare this down to just `CELO`, `mCUSD`, and `WETH`

Reply:
Yeah this particular part needs to be carefully considered as it increases our Sampling search space exponentially. Only use VERY liquid pairs here, which are typically the Native asset and stables.